src/request/request.ts

```diff
@@ -56,10 +56,19 @@ export class Request implements IRequest {
     return undefined
   }
 
-  public header(key: string): string {
+  public header(key: string): string | undefined {
     if (typeof this.event.headers[key] !== 'undefined') {
       return this.event.headers[key]
     }
+    
+    return undefined
+  }
+  
+  public headerOrFail(key: string): string {
+    const value = this.header(key)
+    if (value) {
```

Comment:
lets be verbose maybe? `value !== undefined`

Reply:
false and empty string could still be valid values, right

Reply:
totally agree @zRosenthal great point. @shametim please make changes